missilewars-plugin/src/main/java/de/butzlabben/missilewars/wrapper/abstracts/arena/FallProtectionConfiguration.java

```diff
@@ -30,4 +30,5 @@
 
     private final boolean enabled = true;
     private final int duration = 60;
+    private final boolean message_only_on_start = false;
```

Comment:
Please use the Java convention for variable naming (camelCase). You can modify the serialized name with the annotation
@SerializedName("message_only_on_start")